fluent-tests/src/test/java/com/azure/mgmttest/RuntimeTests.java

```diff
@@ -148,6 +149,13 @@ public void testPom() throws ParserConfigurationException, IOException, SAXExcep
         Assertions.assertTrue(rootTags.get("name").contains("Azure SDK"));
     }
 
+    @Test
+    public void testBotservice(){
+        String siteName = "testSiteName";
+        Site site = new Site().withSiteName(siteName);
+        Assertions.assertEquals(siteName, site.siteName());
+    }
```

Comment:
recommend to do a serialization and de-serialization as well.

Reply:
Ok.

Reply:
Added.